templates/userRoles.tpl

```diff
@@ -0,0 +1,31 @@
+{extends file="layout.tpl"}
+{block name=title}{"User Roles"|_}{/block}
+{block name="content"}
+<h2> {"Users with roles:"|_} </h2>
+<ul>
+  {foreach from=$userRoles item=uRole}
+  <li>{User::get_by_id($uRole->userId)->username}: {Role::get_by_id($uRole->roleId)->name} 
+    <a href="userRoles.php?delete={$uRole->id}" class="btn btn-danger btn-sm">
+    <i class="glyphicon glyphicon-trash"></i>
+    {"delete"|_}
+    </a>
+  </li>
+  {/foreach}
+</ul>
+<h2> {"Assign a role to a user:"|_} </h2>
+<form class="form-inline">
+  <div class="form-group">
+    <label>Username</label>
```

Comment:
E bine ca `<label>` să aibă un atribut `for="username"`, astfel încât câmpul să primească focusul la click pe label. Chestie de accessibility.